src/main/java/seedu/address/model/menu/MenuItem.java

```diff
@@ -42,28 +44,32 @@ public Price getPrice() {
         return price;
     }
 
-    //    public Address getAddress() {
-    //        return address;
-    //    }
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public String itemQuantityOrdered() {
+        return String.valueOf(quantity);
+    }
 
     /**
-     * Returns an immutable tag set, which throws {@code UnsupportedOperationException}
-     * if modification is attempted.
+     * Updates the quantity of this menu item ordered.
      */
-    //    public Set<Tag> getTags() {
-    //        return Collections.unmodifiableSet(tags);
-    //    }
+    public void updateMenuItemQuantity(int newQuantity) {
+        int currentQuantity = this.getQuantity();
+        this.quantity = currentQuantity + newQuantity;
```

Comment:
Do not change the quantity of the MenuItem object. Instead, set it to final and return a new object with new quantity.

Reply:
Also, this method could be prone to bugs. What happens if newQuantity is negative and the quantity of the MenuItem becomes negative? Is negative newQuantity allowed? Shouldn't newQuantity refer to the new quantity, rather than being added to the currentQuantity (naming inconsistency)? Do practise defensive programming.